tox.ini

```diff
@@ -16,7 +16,7 @@ deps =
   testtools>=2.5.0
   networkx>=2.5
   stestr
-passenv = RETWORKX_TEST_PRESERVE_IMAGES
+passenv = RETWORKX_TEST_PRESERVE_IMAGES RESEAUX_PKG_NAME
```

Comment:
We could use [`tox-run-before`](https://pypi.org/project/tox-run-before/) to setup `rustworkx` and `retworkx` in the same environment:

```ini
run_before =
    export RESEAUX_PKG_NAME="rustworkx'
    pip install .
    unset RESEAUX_PKG_NAME
    export RESEAUX_PKG_NAME="retworkx'
    pip install .
    unset RESEAUX_PKG_NAME
```